platform-cloud/docs/orgs-and-teams/organizations.mdx

```diff
@@ -18,15 +18,21 @@ You can also add external collaborators to an organization.
 3. Enter any other optional fields as needed: **Description**, **Location**, **Website URL**, and **Logo**.
 4. Select **Add**.
 
-### Edit an organization
+### Organization settings 
+
+#### Credits 
+
+The **Credits** view contains the current credit balance available to the organization, and the credits spent in the workspace. Select **Contact us to upgrade** to purchase additional credits for your organization. 
```

Comment:
Can we add what Credits enable you to do, what the different billing elements are, how billing works (could be a separate section) and what behavior is when you run out of credits (ie. what won't work/be available)

Reply:
Note – This could also be on the Seqera Compute page

Reply:
There needs to be a section at the start (above this "Credits" block) saying something like "This page allows owners to manage their organizations. The top of the page displays metadata associated with your organization (name, link, location, description)."

Re. Credits: are credits consumed more rapidly if you have existing data in object storage and you're moving data across regions (either ingress or egress)? I would assume yes. If so, we should be explicit.

Definitely needs a more filled-out "what happens when you run out of credits" section to manage expectations (i.e. will someone from Seqera call you, will you be directly to a portal where you enter credit card details, etc?). Might need to be on a different page.